package.json

```diff
@@ -74,6 +74,7 @@
     "@pinata/sdk": "^1.1.23",
     "@polkadot/dev": "^0.62.43",
     "@polkadot/ts": "^0.4.1",
+    "@substrate/connect": "0.3.10",
```

Comment:
```suggestion
```

Added it in https://github.com/polkadot-js/apps/pull/5644 (have too many breakages each time there is a merge...)